src/libraries/System.Net.Security/tests/FunctionalTests/TestConfiguration.cs

```diff
@@ -16,7 +16,7 @@ namespace System.Net.Security.Tests
     internal static class TestConfiguration
     {
         public const int PassingTestTimeoutMilliseconds = 4 * 60 * 1000;
-        public const int FailingTestTimeoutMiliseconds = 250;
+        public const int FailingTestTimeoutMiliseconds = 500;
```

Comment:
In the description you stated:  _"I noticed that the test failed with timeout after ~1s."_

Should we instead increase the timeout for that specific test and move to outerloop instead?

Reply:
It its not that simple as the const is used inside of a helper used by many tests. I made more updates and I think all the tests should run reasonably fast now.

Reply:
Sounds good.